backend/src/jv2backend/journalRoutes.py

```diff
@@ -0,0 +1,187 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2023 Team JournalViewer and contributors
+
+"""Defines the Flask endpoints that only access journal information"""
+import logging
+from flask import Flask, jsonify, request
+from flask.wrappers import Response as FlaskResponse
+
+from jv2backend.journalClasses import JournalLibrary, JournalCollection
+from jv2backend.io.journals.networkLocator import NetworkJournalLocator
+from jv2backend.utils import json_response, split, url_join
+
+
+def add_routes(
+    app: Flask,
+    networkJournalLocator: NetworkJournalLocator,
+    journalLibrary: JournalLibrary
+) -> Flask:
+    """Add routes to the given Flask application."""
+
+    # ---------------- Queries ------------------
```

Comment:
It would be good to include error handling in your routes, for example verifying that the post request contains a body 
e.g:
```
if not request.json:
     abort(400)
```